### PR TITLE
efRuntime

### PR DESCRIPTION
**Pull request by DvirDukhan**

### CODE REVIEW

**Review by ashtul — COMMENTED** [review]
Looks good with a minor question -
In our discussion, I understood we want to set efRuntime at query time. Here we set it at init time.
Do we plan to add query time param later?

**Review by ashtul — CHANGES_REQUESTED** [review]
@DvirDukhan  Please make the clang-format / deletion PR in a separate PR.
There are meaningful changes here and it gets lost in all the formatting changes.

**Comment by DvirDukhan** [comment]
@ashtul done

File: src/VecSim/vecsim.h [hunk 1]
```diff
@@ -7,34 +7,79 @@
...
+            size_t efRuntime;      // EF parameter for HNSW graph accuracy/latency for search.
+            size_t levels;         // Number of graph levels.
+        } hnswInfo;
+    };
+    VecSimType type; // Datatype the index holds.
+    size_t d;        // Vector size (dimension).
+    VecSimAlgo algo; // Algorithm being used.
+    // TODO:
+    // size_t memory;
+} VecSimIndexInfo;
```
Comment (ashtul) [inline src/VecSim/vecsim.h:82]: new line

**Review by ashtul — APPROVED** [review]
LGTM with one minor comment :+1:

**Pull request merged**